docs-ref-services/latest/servicebus-readme.md

```diff
@@ -495,30 +494,30 @@ For more information see the [Code of Conduct FAQ](https://opensource.microsoft.
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
 <!-- LINKS -->
-[azure_cli]: https://docs.microsoft.com/cli/azure
+[azure_cli]: /cli/azure
 [api_docs]: https://azuresdkdocs.blob.core.windows.net/$web/python/azure-servicebus/latest/index.html
-[product_docs]: https://docs.microsoft.com/azure/service-bus-messaging/
+[product_docs]: /azure/service-bus-messaging/
 [azure_portal]: https://portal.azure.com
 [azure_sub]: https://azure.microsoft.com/free/
-[cloud_shell]: https://docs.microsoft.com/azure/cloud-shell/overview
```

Comment:
This readme is sourced from the _actual_ readme in the docs repo.

I will accept these changes, but understand that the next time this package releases a GA version of the package, these updates will be flattened.